### DISCUSSION TITLE
Welcome to cycle2 Discussions!

### DISCUSSION THREAD

**Discussion by iam-sysop** [thecarnie/cycle2#1] [General]
## 👋 Welcome!
  I have added the Discussions feature to the repository to help make it easier to communicate directly any notes about this fork of the Cycle2 project by Mike Alsup.  

  If you have issues with this fork, please continue to use the [Issues ](https://github.com/thecarnie/cycle2/issues) reporting system.
  
  This area can be reserved for casual discussion regarding my efforts on the project as well as notes regarding my progress in maintaining this fork.

**Comment by petertwise** [comment]
Thanks for picking up the torch on this library. We still find it very useful!
